mpxe/harness/pm.py

```diff
@@ -81,7 +80,7 @@ def handle_poll_res(fd, event):
                 for fd, event in res_list:
                     handle_poll_res(fd, event)
         except InvalidPollError as e:
-            return
+            return e
```

Comment:
Returning `e` here is rather confusing since `poll` runs in a thread and its return value isn't used. IMO a better way to fix the pylint warning would be to remove the `as e` part entirely: `except InvalidPollError:` will catch InvalidPollErrors but won't assign a name.